RecoMET/METFilters/python/metFilters_cff.py

```diff
@@ -113,6 +114,7 @@
    chargedHadronTrackResolutionFilter *
    BadChargedCandidateFilter*
    BadPFMuonFilter *
+#  hfNoisyHitsFilter *
```

Comment:
@lathomas 
do I understand correctly that the combined `Flag_METFilters` is not supported/recommended anymore?
If so, the solution here of not backporting the addition of `hfNoisyHitsFilter` would be fine.

Please clarify/confirm.
Thank you.

Reply:
Yes that's correct. Independently of this HF noise filter, the current list is outdated anyways and this flag should not be used. 
We are still discussing what list to make as default (which could possibly include the badpfmuondzfilter from https://github.com/cms-sw/cmssw/pull/31087 ). Once we have converged, we will make a new PR for both 106X and master.